libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt

```diff
@@ -13,6 +14,7 @@ class MapboxDirectionsSession(
 
     private val fasterRouteInterval = 2 * 60 * 1000L // 2 minutes
```

Comment:
This should be customizable from a client point of view as well as enabling / disabling faster functionality completely.

Reply:
Coming up as a part of [this](https://github.com/mapbox/navigation-sdks/issues/221)